doc/api/fs.md

```diff
@@ -1007,6 +1007,33 @@ property indicating whether parent directories should be created. Calling
 `fsPromises.mkdir()` when `path` is a directory that exists results in a
 rejection only when `recursive` is false.
 
+```mjs
+import { mkdir } from 'fs/promises';
+
+try {
+  const path = new URL('./test/project', import.meta.url);
+  const createDir = await mkdir(path, { recursive: true });
+
+  console.log (`created ${createDir}`);
+} catch (err) {
+  console.error(err.message);
+}
+```
+
+```cjs
+const { mkdir } = require('fs/promises');
+const { resolve } = require('path');
+
+async function makeDirectory () {
+  const path = resolve('./test/project/lol/hi')
+  const dirCreation = await mkdir(path, { recursive: true });
+  
+  console.log(dirCreation)
+  return dirCreation;
+}
```

Comment:
```suggestion
async function makeDirectory() {
  const path = resolve('./test/project/lol/hi');
  const dirCreation = await mkdir(path, { recursive: true });
  
  console.log(dirCreation);
  return dirCreation;
}
```

Reply:
Other `async function`s examples in this file have `try`/`catch` blocks, probably this one should to.

https://github.com/nodejs/node/blob/ced518c61a3c362f8870e78aa0a2a08a6ce007c2/doc/api/fs.md#L56-L63
https://github.com/nodejs/node/blob/ced518c61a3c362f8870e78aa0a2a08a6ce007c2/doc/api/fs.md#L6871-L6879